Std/Data/AssocList.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2019 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Leonardo de Moura, Mario Carneiro
 -/
-import Std.Data.List.Basic
```

Comment:
Can you revert this and move the relevant lemmas to `Std.Data.List.Init.Lemmas`?

Reply:
Done.